README.md

```
@@ -81,3 +77,22 @@ dx run eggd_plot_variant_baf \
 -ipackages=file-xxxx  \
 --destination="output/eggd_plot_variant_baf"
 ```
+With all possible input parameters:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)